lnrpc/walletrpc/errors.go

```diff
@@ -0,0 +1,17 @@
+package walletrpc
+
+import "errors"
+
```

Comment:
Since `walletkit_server.go` is dependent on build tags the errors defined in it aren't always available, e.g. during unit test runs. Hence I moved them to a separate file. Is there a better way to make `ErrInsuficientReserve` available in `lnd_onchain_test.go`?

Reply:
I think it would've been okay to keep the errors where they are. Because you can only access the RPC from the itest if the build tag is used, it's okay to also use variables guarded behind build tags. But since you already made the change, let's keep them in the separate file.